development/workflow.rst

```diff
@@ -0,0 +1,305 @@
+A common pattern is:
+
+.. code-block:: bash
+
+   git checkout master
+   git pull
+   git checkout tickets/DM-NNNN
+   git rebase -i master
+   # interactive rebase
+   git push --force
+
+Use `Jenkins at ci.lsst.codes <https://ci.lsst.codes/job/stack-os-matrix/build?delay=0sec>`_ to run the Stack's tests with your ticket branch work.
+With Jenkins you can specify the name (or names) of the ticket branches in repositories you are developing; Jenkins will test your work against the ``master`` branch of other Stack repositories by default.
+
+You can monitor builds in the “Bot: Jenkins” HipChat room.
+
+Assign a reviewer
+-----------------
+
+On GitHub, `create a pull request <https://help.github.com/articles/creating-a-pull-request/>`_ for your ticket branch.
+
+.. FIXME policy for naming pull requests?
+
+On your ticket's JIRA page, use the **Workflow** button to switch the ticket's state to **In Review**.
+
+JIRA will ask you to assign reviewers.
+Good reviewers might be experts in the package or domain your are developing.
+But also don't overload a small group of people with code reviews.
+It's also a good idea to distribute your review assignments across the collaboration.
+In particular, asking junior team members to review code is a good way to have them broaden their knowledge of the LSST Stack.
+
+In your message to reviewers, indicate how involved the review work will be ("quick" or a "not quick").
+
+The code review
+---------------
+
+All code review discussion should happen on the GitHub pull request, *not* on the JIRA ticket.
+Working on the PR allows for code to be referenced.
+Patch-level comments are preserved by GitHub even if new commits are force pushed.
+
+The reviewer should examine the code with DM's best practices in mind.
+
+- Does the code adhere to DM's `Python <https://confluence.lsstcorp.org/display/LDMDG/Python+Coding+Standard>`_ and `C++ <https://confluence.lsstcorp.org/pages/viewpage.action?pageId=16908666>`_ style guides?
+- Is there unit test coverage?
+- Are :doc:`Python Docstrings </development/docs/py_docs>` and :doc:`C++ Doxygen </development/docs/cpp_docs>` comments complete? Check the locally-built HTML documentation for your package to ensure the formatting is correct.
+- Are the code additions or changes reflected in the documentation, tutorials and examples?
+- Are the Git commits :ref:`well organized <git-commit-organization-best-practices>` and are the :ref:`commit messages useful <git-commit-message-best-practices>`?
+
+The reviewer should also consider the work within the larger context of the LSST Stack.
+Does the contribution fit in the LSST Stack's architecture?
+
+The review process will likely trigger new commits.
+Following :ref:`our best practices <git-commit-organization-best-practices>`, any fix-up commits should be squashed into the appropriate original commit to present a clean development story.
+
```

Comment:
I don't think there's anything in that link about authority in a review, and I've never seen any DM document or e-mail mention it.